src/lib/DatePicker.svelte

```diff
@@ -427,4 +412,13 @@
       color: var(--date-picker-selected-color, inherit)
       background: var(--date-picker-selected-background, rgba(2, 105, 247, 0.2))
       border: 2px solid var(--date-picker-highlight-border, #0269f7)
+
+  .calendar-day:focus
+    outline: none
+  .calendar-day:focus-visible
```

Comment:
This can be `:focus` instead of `:focus-visible` because I added a workaround for it

Reply:
fixed.